### PR TITLE
Release smart_proxy_remote_execution_ssh 0.0.6 (RPM)

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec [hunk 1]
```diff
@@ -14,7 +14,7 @@ URL: https://github.com/theforeman/smart_proxy_remote_execution_ssh
 Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
 Requires: foreman-proxy >= 1.9.0
-Requires: rubygem(smart_proxy_dynflow) >= 0.0.3
+Requires: rubygem(smart_proxy_dynflow) >= 0.0.4
```
Comment (domcleal) [inline rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec]: This is still 0.0.3

Comment (stbenjam) [inline rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec]: https://github.com/theforeman/foreman-packaging/pull/818 ?

Comment (stbenjam) [inline rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec]: Oh I see, from the gemspec

Comment (domcleal) [inline rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec]: I mean the dependency in the gem is still 0.0.3.

Comment (stbenjam) [inline rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec]: Fixed

**Comment by domcleal** [comment]
Thanks @stbenjam, building.

**Pull request merged**